debug-bar.php

```diff
@@ -30,6 +20,8 @@
 		return;
 	}
 
+	_deprecated_function( 'debug_bar_enable', '1.1.3', 'wpcom_vip_qm_enable' );
```

Comment:
Unfortunately won't end up in logs. Need to `trigger_error( $message, E_USER_WARNING )` if we want to be able to look for the notice / expose to customers.